VAMobile/src/components/Templates/LargePanel.tsx

```diff
@@ -128,7 +132,7 @@ export const LargePanel: FC<LargePanelProps> = ({
   }
 
   return (
-    <>
+    <Modal visible={!hideModal} transparent={true}>
```

Comment:
Can you add a comment about why this modal wrapping is necessary and what it fixes? Keyboard nav is going to be really easy to lose track of when people have reasons to fix other issues by removing something like this modal